PixiEditor.ChangeableDocument/Changes/UpdateableChange.cs

```diff
@@ -2,5 +2,10 @@
 
 internal abstract class UpdateableChange : Change
 {
+    /// <summary>
+    /// Applies the change temporarily
```

Comment:
'ApplyTemporarily' is kind of a bad name, and the doc only solidifies it. For example, if you look at how pens are implemented you'll notice that ApplyTemporarily is used to draw a line to the mouse position. The old lines don't get erased. Really this method is just "`Update`, the sequel" but idk what a good name for it would be